labeler/has_pull_request.go

```diff
@@ -0,0 +1,47 @@
+package labeler
+
+import (
+	"log"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/github"
+	"github.com/parkr/auto-reply/ctx"
+)
+
+var fixesIssueMatcher = regexp.MustCompile(`Fixes #(\d+)`)
```

Comment:
A list of keywords can be found at https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues